pkg/restapi/common/store/tokens/store.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package tokens
+
+import (
+	"fmt"
+	"github.com/trustbloc/edge-agent/pkg/restapi/common/store"
+	"github.com/trustbloc/edge-core/pkg/storage"
+)
+
+const (
+	// StoreName is the name of the token store.
+	StoreName = "edgeagent_tks"
+)
+
+// UserTokens are the tokens associated to a User.
+type UserTokens struct {
```

Comment:
can that structure hold multiple tokens?

Reply:
@aholovko It already is (access_token, refresh_token).